### ISSUE TITLE
[ru] Запятая в сложносочинённом предложении

### ISSUE THREAD

**Issue by kostyfisik** [languagetool-org/languagetool#1082]
WRONG
Если его направление отличается от текущего на значение большее, чем пороговый угол, то длина отрезка уменьшается в два раза и расчёт производится заново до тех пор, пока не будет найдено удовлетворительное значение длины.
CORRECT
Если его направление отличается от текущего на значение большее, чем пороговый угол, то длина отрезка уменьшается в два раза, и расчёт производится заново до тех пор, пока не будет найдено удовлетворительное значение длины.

**Comment by srcmilena** [comment]
Hello, Konstantin!

Would let me know if this problem is solved? So we can update its status.

**Comment by kostyfisik** [comment]
No, it is not. It still fails to find an error at https://languagetool.org/ru/
